app/components/github-repo.js

```diff
@@ -3,20 +3,46 @@ import recordsList from 'code-corps-ember/utils/records-list';
 
 const {
   computed,
+  computed: { alias, filter, mapBy, notEmpty },
   Component,
-  get
+  get,
+  inject: { service }
 } = Ember;
 
 export default Component.extend({
   classNames: ['github-repo'],
+  classNameBindings: ['isConnected:github-repo--connected'],
   tagName: 'li',
 
-  isConnected: computed('githubRepo', 'project.projectGithubRepos.@each', function() {
-    let githubRepo = get(this, 'githubRepo');
-    let project = get(this, 'project');
-    let projectGithubRepos = get(this, 'project.projectGithubRepos');
-    if (projectGithubRepos) {
-      return recordsList.includes(projectGithubRepos, githubRepo);
+  githubRepo: null,
+  project: null,
+
+  store: service(),
+
+  isConnected: notEmpty('projectsGithubRepo'),
+  projectGithubRepo: computed('projectGithubRepos.@each', 'projectGithubRepos.isFulfilled', 'githubRepo.id', function() {
```

Comment:
This is the part that always bugs me. I'm not sure we need to watch `isFulfilled` here. I'm not even sure we need to watch `@each`. Possibly even just the base is enough, but I'm not sure.